_posts/remembering-mary.md

```diff
@@ -11,14 +11,16 @@ ogImage:
 banner: 
   text: "Remembering Mary - A Friend, Advocate, and Champion of DATA. Your passion and commitment will always be remembered."
 ---
-Yesterday, we found out that Mary Collier, our Events Planning Committee Chair passed away on Tuesday.
+Yesterday (January 24, 2025), we found out that Mary Collier, our Events Planning Committee Chair passed away on Tuesday (January 21, 2025).
 
 Mary was a fierce warrior for public transportation while also planning so many social gatherings that brought this community together.
 
-Mary has several planned events in the [calendar](/calendar), and I realize that may raise some important questions. The Executive Committee has decided that out of respect for her and all the work she put into DATA, those events will not be cancelled. It is our belief that she would have wanted us to keep the community together and continue our efforts to save DART, so that is exactly what we will be doing.
+Mary has several planned events in the [calendar](/calendar), and we realize that may raise some important questions. The Executive Committee has decided that out of respect for her and all the work she put into DATA, those events will not be cancelled. It is our belief that she would have wanted us to keep the community together and continue our efforts to save DART, so that is exactly what we will be doing.
```

Comment:
```suggestion
Mary had planned several events in the [calendar](/calendar), and we realize that may raise some important questions. The Executive Committee has decided that out of respect for her and all the work she put into DATA, those events will not be cancelled. It's our belief that she would have wanted us to keep the community together and continue our efforts to save DART, so that's exactly what we will be doing.
```